api/XRAnchor.json

```diff
@@ -0,0 +1,145 @@
+{
+  "api": {
+    "XRAnchor": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": "85"
+          },
+          "chrome_android": {
+            "version_added": "85"
+          },
+          "edge": {
+            "version_added": "85"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
```

Comment:
Looks like there's both a CG (https://www.w3.org/community/immersive-web/) and. a WG (https://www.w3.org/immersive-web/) for this, so I guess `standards_track` true is appropriate here.

But `experimental` is set to true for some of what already exists in api/XRFrame.json, but not all existing XR* stuff it seems.

Can you go over and check what's most common and make sure we set the same statuses for all of XR* unless there's nothing differentiating them? I'd default to `experimental` true and `standards_track` true if the existing pattern is not clear.